app/services/payment_provider_customers/stripe_service.rb

```diff
@@ -84,7 +84,7 @@ def generate_checkout_url(send_webhook: true)
       result
     rescue ::Stripe::InvalidRequestError, ::Stripe::PermissionError => e
       deliver_error_webhook(e)
-      result
+      fail! e
```

Comment:
Oups... Indeed, this one in this scope should not be swallowed... :+1:

Reply:
I'll fix this separately so we can get it fixed today